### DISCUSSION TITLE
Elasticity parameter for collisions

### DISCUSSION THREAD

**Discussion by onnoeberhard** [google/brax#374] [Q&A]
Dear Brax community, is it possible to control the percentage of kinetic energy that is dissipated in a collision? I have noticed that the Ant, when all torques are set to a maximum such that it is very stiff, bounces as though the collisions with the ground are nearly elastic. It would be great to set such a parameter between 0 and 1 to make the collisions inelastic.

**Comment by btaba** [comment]
You can control elasticity depending on the physics backend used in brax. For the `positional` pipeline, you can use elasticity, as in https://github.com/google/brax/blob/ac2b94bad71658a2cc194518c6585c11117663fe/brax/test_data/capsule.xml#L7

For the `generalized` pipeline you can use the constraint solver parameters `solref` as described in https://mujoco.readthedocs.io/en/latest/modeling.html#solver-parameters. These control the stifness and damping associated with the collision constraint
